### DISCUSSION TITLE
[resolved] Historia statusów przesyłek z "wysyłam z Allegro" dla przesyłek utworzonych ręcznie

### DISCUSSION THREAD

**Discussion by luiggies** [allegro/allegro-api#4939] [Suggestions - resolved]
Jak pobrać historię statusów przesyłek z "wysyłam z Allegro" dla przesyłek nie utworzonych przez API a ręcznie przez stronę?
korzystam z 
GET /order/carriers/ALLEGRO/tracking?waybill={waybill} lecz gdy podaję numer przesyłki wywala błąd
waybill'a pobieram za pomocą:
GET /order/checkout-forms/{id_zamówienia}/shipments
i jest on zbieżny z numerem przesyłki dostępnym w wysylam.allegro.pl

**Comment by PrzemyslawLukanowski** [comment]
Aktualnie przez ten zasób możesz pobrać statusy tylko dla przesyłek Allegro (czyli nadanych metodą dostawy Allegro Punkty Kolporter). Zgłoszenie zostawiam jako sugestię dla przesyłek nadanych inną metodą dostawy.

**Reply by luiggies** [reply]
Czy istnieje jakikolwiek inny sposób by sprawdzić historię albo chociż status przesyłki w API allegro czy standardowo należ sprawdzać to tylko w dedykowanym API firm kurierskich?

**Reply by PrzemyslawLukanowski** [reply]
Na tę chwilę jedyny sposób to dedykowane API firm kurierskich. Opublikujemy odpowiedni komunikat, jeśli coś się zmieni w tym temacie.

**Reply by luiggies** [reply]
Ok, będzie miło jak będzie taki zasób, pozdrawiam

**Reply by Radeq** [reply]
Zgadzam się z @luiggies  - informacje o statusie przesyłki dla pozostałych kurierów byłyby bardzo przydatne. API innych brokerów kurierskich zwracają ustandaryzowane statusy.
